openwisp_users/api/urls.py

```diff
@@ -9,6 +10,16 @@ def get_api_urls(api_views=None):
     urlpatterns = []
     if api_views is None:
         api_views = views
+    urlpatterns += [
+        path('user/organization/', views.organization_list, name='organization_list',),
+        path(
+            'user/organization/<str:pk>/',
+            views.organization_detail,
+            name='organization_detail',
+        ),
+        path('user/users/', views.users_list, name='user_list',),
```

Comment:
@nemesisdesign , hey, what do you think about the endpoints `/user/organization/` and `/user/users`?
I would personally prefer something other than that `/user/users`, but what would your advice be?

Reply:
in general we keep resource name in the singluar form, there are some exceptions but not in this case.

If we wanted to prefix the module, we should rather use users/ from openwisp-users.

So it could be: /users/user/, /users/organization/, /users/group/, etc.

If we want to keep consistency with what we are doing in the other modules, this is probably the best we can do, do you have any alterantive in mind?

Reply:
perhaps `/iam/user` or `/identity/user`; `users/user`, the same word repeated just looks odd, but it works as well!

Reply:
iam and identity are two words we don't use at all